app/App/Panel/Main/Account/Requests/TransactionRequest/index.js

```diff
@@ -419,7 +415,23 @@ class TransactionRequest extends React.Component {
         )}
         {!notice ? (
           <div className='requestApprove'>
-            {req.recentFeeUpdate ? <div className='requestApproveBlock cardShow'>{'transaction fee updated'}</div> : null}
+            {req.automaticFeeUpdateNotice ? (
+              <div className='requestApproveFeeBlock cardShow'>
+                <div className='requestApproveFeeButton requestApproveFeeReject' onClick={() => {
+                  const { previousFee } = req.automaticFeeUpdateNotice
```

Comment:
would `previousTx` or something like that be more descriptive here?

Reply:
`previousFee` allows users to accept/rollback automatic fee updates, it only contains previous fee data for the current tx so we can rollback if needed